themes/openy_themes/openy_carnation/webpack.build.js

```diff
@@ -0,0 +1,9 @@
+const merge = require('webpack-merge');
+const common = require('./webpack.common.js');
+
+module.exports = merge(common, {
+  mode: 'production',
+  optimization: {
+    minimize: true,
```

Comment:
because we are doing minification, let's ensure in openy_carnation.libraries all js files have `minified:true` for not facing with aggregation issues
see https://github.com/ymcatwincities/openy/blob/8.x-2.x/themes/openy_themes/openy_lily/openy_lily.libraries.yml#L13

Reply:
Hi. At your example only CDN-loaded libraries are marked as minified.
Why do we need to add minified: true to our theme libraries?

Reply:
Because you do minification for them, aren't you?

Reply:
what this `'minimize': true` setting drives?

Reply:
We are not including those scripts to webpack prepared scripts.
We assuming that Drupal aggregation and minification will do this

Reply:
still confusing